mac/mac.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "libmac",
+  "no-autogen": true,
+  "rm-configure": true,
+  "make-install-args": [
+    "prefix=/app"
+  ],
+  "cleanup": [
+    "/bin/mac"
+  ],
```

Comment:
Not sure if this cleanup should be included though. Since we try to have a generic package here, some applications might need the main binary to use mac.

**Edit.** Can confirm. Removing "/bin/mac" actually breaks mac support in Asunder. @enzo1982 are you OK with removing the cleanup?

Reply:
You're right, of course. I removed that command.